api/bin/phpunit

```diff
@@ -0,0 +1,23 @@
+#!/usr/bin/env php
```

Comment:
Do we really need PHPUnit for the API? Maybe could we remove all unnecessary packages to make the maintenance easier?